01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,39 @@
+const { writeFile, readFile } = require("fs").promises;
+
+const filePath = "./temporary/temp.txt";
+
+const writer = async () => {
+  try {
+    const first = await readFile("./content/first.txt", "utf8");
```

Comment:
This is a logic error, your write should on write File, not read. Please read the directions carefully and feel free to discuss with other students.


 await writeFile("temp.txt","Line 1");
  await writeFile("temp.txt","Line 2\n", {flag: "a"});
   var test = await writeFile("temp.txt","Line 3", {flag: "a"});
   return test // where test would be a promise